modules/appeals_api/spec/factories/evidence_submissions.rb

```diff
@@ -6,13 +6,18 @@
     guid { SecureRandom.uuid }
     association :supportable, factory: :notice_of_disagreement
     upload_submission { create(:upload_submission, guid: SecureRandom.uuid) } # set the guid to pass uniqueness check
+  end
 
-    trait :with_detail do
```

Comment:
This trait is failing the Migration PR. You may want to remove this in it's own PR, and get that merged ahead of time.

Reply:
I was going to merge this and then the migration. Should I not wait?

Reply:
I think since the migration is just removing columns, that should be fine.